docs/setup/install.md

```diff
@@ -0,0 +1,82 @@
+# Installation
+
+## Before you begin
+
+Make sure the following conditions are met:
+
+- A Kubernetes cluster with version 1.21 or newer is running. Learn how to [install the Kubernetes tools](https://kubernetes.io/docs/tasks/tools/).
```

Comment:
This should be 1.26 at least in this release we enabled the `Start ordinal` see, https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#start-ordinal, also mention that we should enable the feature gate under 1.27.

Reply:
+1

Reply:
sg!